pkg/netpol/connlist/connlist.go

```diff
@@ -44,18 +44,25 @@ import (
 // A ConnlistAnalyzer provides API to recursively scan a directory for Kubernetes resources including network policies,
 // and get the list of permitted connectivity between the workloads of the K8s application managed in this directory.
 type ConnlistAnalyzer struct {
-	logger           logger.Logger
-	stopOnError      bool
-	errors           []ConnlistError
-	focusWorkloads   []string
-	focusDirection   string
-	exposureAnalysis bool
-	exposureResult   []ExposedPeer
-	explain          bool
-	outputFormat     string
-	muteErrsAndWarns bool
-	peersList        []Peer // internally used peersList used in dot formatting; in case of focusWorkload option contains only relevant peers
-}
+	logger             logger.Logger
+	stopOnError        bool
+	errors             []ConnlistError
+	focusWorkloads     []string
+	focusWorkloadPeers []string
+	focusDirection     string
+	exposureAnalysis   bool
+	exposureResult     []ExposedPeer
+	explain            bool
+	outputFormat       string
+	muteErrsAndWarns   bool
+	peersList          []Peer // internally used peersList used in dot formatting; in case of focusWorkload option contains only relevant peers
+	ignoreExposure     bool   // internally used; indicates if exposure flag is used while it has no effect (to be ignored)
```

Comment:
why not changing `ca.exposureAnalysis` to false in case it should be ignored?

Reply:
done